src/controller/age.controller.js

```diff
@@ -0,0 +1,37 @@
+const { Router } = require("express");
+const router = Router();
+/**
+ * @description Calculate user age
+ * @returns {number}
+ */
+
+router.get("/howold", async (req, res) => {
+  const { dob } = req.query;
+
+  if (!dob || dob == "") {
+    return res.status(400).json({
+      error: "Please provide a date of birth",
+    });
+  }
+
+   const today = new Date();
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
  const today = new Date();
```